R/format_MTA.R

```diff
@@ -0,0 +1,371 @@
+#' Construct standard format data from MTA-PE Evolutionary Ecology Group, University of Pannonia, Hungary (MTA)
+#'
+#' A pipeline to produce the standard format for bird study population
+#' at the MTA-PE Evolutionary Ecology Group, Hungary, administered by
+#' Gabor Seres & András Liker.
+#' The data include 5 sites: Veszprém, Balatonfüred, Szentgál, Vilma-puszta and Gulya-domb.
+#'
+#' This section provides details on data management choices that are unique to
+#' this data. For a general description of the standard format please see see
+#'\href{https://github.com/SPI-Birds/documentation/blob/master/standard_protocol/SPI_Birds_Protocol_v1.0.0.pdf}{here}.
+#'
+#' \strong{xxx}:
+#'
+#' \strong{Latitude, Longitude} The exact coordinates of the nestboxes are not available.
+#' Data owner provided an map of the location, which can be georeferenced in the case of
+#' interest or necessity of the data user. Currently, only general coordinates
+#' for the location are provided.
+#'
+#' @inheritParams pipeline_params
+#' @return Generates either 4 .csv files or 4 data frames in the standard format.
+#' @export
+
+
+#### --------------------------------------------------------------------------~
+#### QUESTIONS DATA OWNERS
```

Comment:
Antica's comment from the SPI-Birds spreadsheet:

"their data is in hungarian, but my understanding is that he has sent a translated version.... check before working on this"

Should check with data owner when you contact them.

Reply:
Yes, I have seen that Antica's comment. The only word in Hungarian I found is the "gyurutlen" which based on dictionary refers to unringed bird. Appears only several times, majority of unringed individuals are labeled "ringless" or "unringed" in English.